### PR TITLE
Kostanyan Arsen lab1, var1 (patch)

### PR DESCRIPTION
**Pull request by arkostaa**
Made it so it does not output Exit code 1 with --help.

### CODE REVIEW

File: clang/test/lab1/kostanyan_arsen/tests.cpp [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+class Test4 {
+  class Nested {
+    int z;
+  };
+};
+// CHECK: Test4
+// CHECK: Nested
+// CHECK-NEXT: |_z
+
+
```
Comment (m-ly4) [inline clang/test/lab1/kostanyan_arsen/tests.cpp]: You made first part. Help option doesn't crash a compiler. But your plugin does its job even if 'help' is specified, whilst it shouldn't.
Please try to add the same help test into your previous test file:
```suggestion
// RUN: %clang_cc1 -load %llvmshlibdir/PrintNamesPlugin%pluginext -plugin classprinter -plugin-arg-classprinter --help %s 2>&1 | FileCheck %s --check-prefix=HELP
// HELP: This plugin traverses the Abstract Syntax Tree (AST) of a codebase and prints the name and fields of each class it encounters
// HELP-NOT: |_
```

**Review by m-ly4 — CHANGES_REQUESTED** [review]

File: clang/test/lab1/kostanyan_arsen/tests.cpp (hunk 1 above)
Comment (arkostaa) [reply clang/test/lab1/kostanyan_arsen/tests.cpp]: I've changed it now

**Review by m-ly4 — APPROVED** [review]

File: clang/test/lab1/kostanyan_arsen/tests.cpp (hunk 1 above)
Comment (m-ly4) [reply clang/test/lab1/kostanyan_arsen/tests.cpp]: Thank you.

**Review by m-ly4 — COMMENTED** [review]
Could you please fix clang format?

**Pull request closed without merging**